lib/index.js

```diff
@@ -82,18 +86,24 @@ Optimizely.prototype.initialize = function() {
  */
 
 Optimizely.prototype.track = function(track) {
-  var payload = {};
+  // Use the new-style API (which is compatible with Classic and X)
+  var payload = {
+    type: 'event',
+    eventName: track.event(),
+    tags: track.properties()
+  };
   var revenue = track.revenue();
```

Comment:
update: oops nevermind I see you are just converting the `tags.revenue` to cents.